manifest.yml

```diff
@@ -1,13 +1,18 @@
-image: adashen/iot-edge-testing-utility:0.0.1
-tags: [0.0.1-amd64, 0.0.1-windows-amd64]
+image: aziotedgetooling.azurecr.io/public/azureiotedge-testing-utility:1.0.0
+tags: [1.0.0-amd64, 1.0.0-windows-amd64, 1.0.0-arm32v7]
 manifests:
   -
-    image: adashen/iot-edge-testing-utility:0.0.1-amd64
+    image: aziotedgetooling.azurecr.io/public/azureiotedge-testing-utility:1.0.0-amd64
     platform:
       architecture: amd64
       os: linux
   -
-    image: adashen/iot-edge-testing-utility:0.0.1-windows-amd64
+    image: aziotedgetooling.azurecr.io/public/azureiotedge-testing-utility:1.0.0-windows-amd64
     platform:
       architecture: amd64
-      os: windows
\ No newline at end of file
+      os: windows
+  -
+    image: aziotedgetooling.azurecr.io/public/azureiotedge-testing-utility:1.0.0-arm32v7
```

Comment:
I cannot remember why we didn't publish arm previously. We may need to have a test

Reply:
ok